src/unittest/test.cpp

```diff
@@ -65,6 +65,9 @@ class TestGameDef : public IGameDef {
 		return testmodspec;
 	}
 	virtual const ModSpec* getModSpec(const std::string &modname) const { return NULL; }
+	virtual std::string getModStoragePath() const { return "/tmp"; }
```

Comment:
Not important at the moment but may cause problems for Windows users when running the unittests (once they test the mod storage).